packages/storybook/.storybook/main.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
       include: path.resolve(__dirname, '../'),
     });
 
+    config.resolve.alias['@department-of-veterans-affairs/component-library$'] =
+      path.resolve(__dirname, '../../react-components/src');
```

Comment:
This is so the docs page for each component in storybook can populate the args table. The propTypes are stripped out in the built version, so we need to use the source files.

Reply:
This is worth preserving in a code comment. It's also probably worth calling out in the Storybook package's README so folks later will know there's a bit of misdirection going on here.